Loop/Managers/DeviceDataManager.swift

```diff
@@ -1352,12 +1353,12 @@ extension DeviceDataManager {
 
         switch command {
             
```

Comment:
For overrides, bolus, carbs -- I moved the logic into dedicated methods given the expanded things that happen.